src/parse/asp/interpreter.go

```diff
@@ -113,7 +113,7 @@ func (i *interpreter) interpretStatements(s *scope, statements []*Statement) (re
 			} else {
 				err = fmt.Errorf("%s", r)
 			}
-			log.Debug("%s", debug.Stack())
+			log.Debug("%v:\n %s", err,  debug.Stack())
```

Comment:
this got me a little confused when reading `build.log`. The err is printed with a log.Error later on but that ends up after the stack trace.